### PR TITLE
Enable coveralls.io integration

### PR DESCRIPTION
**Pull request by detiber**
Coveralls will allow us to better track and visualize our test coverage over time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [![Join the chat at https://gitter.im/openshift/openshift-ansible](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/openshift/openshift-ansible)
 [![Build Status](https://travis-ci.org/openshift/openshift-ansible.svg?branch=master)](https://travis-ci.org/openshift/openshift-ansible)
+[![Coverage Status](https://coveralls.io/repos/github/openshift/openshift-ansible/badge.svg?branch=master)](https://coveralls.io/github/openshift/openshift-ansible?branch=master)
```
Comment (ashcrow) [inline README.md]: Currently shows unknown but I assume that's because it's not enabled yet :smile:

Comment (detiber) [inline README.md]: @ashcrow yes, no tests have been run against master yet :)

**Comment by ashcrow** [comment]
aos-ci-test

**Comment by ashcrow** [comment]
:fireworks:

**Comment by rhcarvalho** [comment]
Let the gamification begin! Extra points to those who make the coverage bar go up? :-D

**Pull request merged**